ChameleonForms/FieldGenerators/Handlers/DateTimeHandler.cs

```diff
@@ -36,7 +37,15 @@ public override IHtmlString GenerateFieldHtml(IReadonlyFieldConfiguration fieldC
         public override void PrepareFieldConfiguration(IFieldConfiguration fieldConfiguration)
         {
             if (!string.IsNullOrEmpty(FieldGenerator.Metadata.DisplayFormatString))
-                fieldConfiguration.Attr("data-val-format", FieldGenerator.Metadata.DisplayFormatString.Replace("{0:", "").Replace("}", ""));
+            {

```

Comment:
1. small 'g' is for combined short date and short time.
2a. yes, such issue is present, but for us it's ok. our web-server explicitly sets UICulture in BeforeRequest according to accept languages of the client or fallbacks to closest one.
2b. as for me js should convert time from client's culture format to unix/filetime and only then this value should be bound to model on the server. but I don't have enough time and experience in js to implement it sufficiently.